FreeRTOS-Plus/Demo/AWS/Mqtt_Demo_Helpers/mqtt_demo_helpers.c

```diff
@@ -118,8 +118,11 @@
 
 /**
  * @brief Timeout for MQTT_ProcessLoop in milliseconds.
```

Comment:
Can we add one more line.
Currently, 15 topics are subscribed and based on experimentation a timeout of 2000ms is necessary to maintain a steady state throughout the demo.
So, the comment will look like this:

/**
 * @brief Timeout for MQTT_ProcessLoop in milliseconds.
 * This timeout corresponds to time taken by Number of topic subscribed to publish
 * messages from device to broker and broker to device to complete.
 * Currently, 15 topics are subscribed and based on experimentation a timeout of 2000ms is necessary
 * to maintain a steady state throughout the demo.
 * Any change in Number of topic subscribed needs updating mqttexamplePROCESS_LOOP_TIMEOUT_MS.
 */